README.md

```diff
@@ -1,6 +1,9 @@
-# xtext-index-backend
+# DEPRECATED xtext-index-backend
 xtext-index-backend
 
+:warning: **WARNING** This project is deprecated because the Test-Editor Team decided to cut off the RCP functionality  for the sake of the Test-Editor-Web in October 2018 :warning:
```

Comment:
Same as with [_test-editor-index-backend_](https://github.com/test-editor/deprecated-test-editor-index-backend/pull/6), this project used to be part of Test-Editor-Web. It used to provide generic Xtext index services (i.e. non-specific to a particular language, meant to also be used by other web-facing projects based on Dropwizard and Xtext).